sdks/python/apache_beam/io/fileio_test.py

```diff
@@ -40,14 +41,16 @@ class MatchTest(_TestCaseWithTempDirCleanUp):
 
   def test_basic_two_files(self):
     files = []
-    tempdir = '%s/' % self._new_tempdir()
+    tempdir = self._new_tempdir()
```

Comment:
Can you use the context manager here ? https://github.com/apache/beam/blob/master/sdks/python/apache_beam/testing/test_utils.py#L41

This is used by other file-based IO tests.